cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -465,4 +773,13 @@ output AZURE_RESOURCE_REDIS_ID string = redis.outputs.resourceId
 {{- if .DbPostgres}}
 output AZURE_RESOURCE_{{alphaSnakeUpper .DbPostgres.DatabaseName}}_ID string = '${postgreServer.outputs.resourceId}/databases/{{.DbPostgres.DatabaseName}}'
 {{- end}}
+{{- if .DbMySql}}
```

Comment:
Likewise, should these also start with `AZURE_RESOURCE_`? How are these outputs used?